scripts/screeps_rl_experiment_card.py

```
@@ -298,9 +525,99 @@ def scenario_supports_multi_tier_policy_comparison(raw: Any) -> bool:
+        ("hostile_spawn_count", "hostileSpawnCount", fixture_summary["adjacentHostileSpawnCount"]),
+        ("own_anchor_spawn_count", "ownAnchorSpawnCount", fixture_summary["anchorOwnSpawnCount"]),
+        ("own_anchor_creep_count", "ownAnchorCreepCount", fixture_summary["anchorOwnCreepCount"]),
+        ("fixture_sha256", "fixtureSha256", fixture_summary["fixtureSha256"]),
+    )
+    for snake_key, camel_key, expected in expected_fields:
+        observed = first_present(evidence, (snake_key, camel_key))
+        if observed != expected:
+            raise error_cls(f"multi-tier scenario evidence.{snake_key} must match the active fixture")
+
+
+def int_or_none(value: Any) -> int:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)